src/frontend-samples/tree-samples/drag-and-drop-tree/index.tsx

```diff
@@ -0,0 +1,40 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
This file should be named DragAndDropTreeUI.tsx to match the convention in the other samples